### PR TITLE
[INLONG-7479][Manager] Forbidden to configure stream when group configuration fails

### LINKED ISSUE #7479: [Bug][Manager] The stream can still be configured under the failed group
### What happened

The stream can still be configured under the failed group.

### What you expected to happen

The stream can still be configured under the failed group.

### How to reproduce

Configure a group workflow.
Re-execute the failed stream.

企业微信截图_edd1820a-2135-40e2-a627-2d3c4b1151fd

企业微信截图_16774687283935

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #7479 

### Motivation

Forbidden to configure stream when group configuration fails.

### Modifications

Forbidden to configure stream when group configuration fails

### CODE REVIEW

**Review by vernedeng — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/queue/StreamQueueResourceListener.java [hunk 1]
```diff
@@ -83,6 +85,9 @@ public ListenerResult listen(WorkflowContext context) throws WorkflowListenerExc
             log.error(msg);
             throw new WorkflowListenerException(msg);
         }
+        GroupStatus groupStatus = GroupStatus.forCode(groupInfo.getStatus());
+        Preconditions.expectTrue(GroupStatus.CONFIG_FAILED != groupStatus,
```
Comment (gosonzhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/queue/StreamQueueResourceListener.java:89]: An error code needs to be carried when an exception is thrown

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java [hunk 2]
```diff
@@ -213,4 +214,16 @@ public boolean delete(String groupId, String streamId) {
...
         return response.getData();
     }
+
+    /**
+     * Converts a json string to a streamFields
+     *
+     * @param fieldsJson JSON string for the field information
+     * @return list of stream field
+     */
+    public List<StreamField> parseFields(String fieldsJson) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java:224]: This method seems not related to this issue.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/queue/StreamQueueResourceListener.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/queue/StreamQueueResourceListener.java:89]: This doesn't need to be set, it doesn't get thrown out.

**Review by gosonzhang — APPROVED** [review]

File: inlong-manager/manager-web/bin/restart.sh [hunk 3]
```diff
@@ -36,10 +36,7 @@ BIN_PATH=$(
 echo "restart in" ${BIN_PATH}
 
 # Stop service
-sh "$BIN_PATH"/shutdown.sh
```
Comment (healchow) [inline inlong-manager/manager-web/bin/restart.sh:39]: Why change this script here?

Comment (dockerzhang) [reply inlong-manager/manager-web/bin/restart.sh:39]: image

Comment (healchow) [reply inlong-manager/manager-web/bin/restart.sh:39]: @fuweng11 Please revert this file's change, thanks.

**Pull request merged**